models/Challenge.js

```diff
@@ -52,6 +60,7 @@ class Challenge {
         resolve(null);
       }
 
+      console.log(user.instrument, user.part, user.spotId, user.nameNumber);
```

Comment:
Debug log? Wouldn't be surprised to see this removed in a later commit that I haven't reviewed yet.